webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -85,11 +85,24 @@ defmodule VideoRoomWeb.RoomChannel do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_info({:signal, {:replace_track, old_track_id, new_track_id}}, socket) do
+    broadcast!(socket, "replace_track", %{
+      data: %{
+        "type" => "replace_track",
+        "old_track_id" => old_track_id,
```

Comment:
And if you send some maps that will be then converted to json rather use camel case for them as javascript does not really like snake case.